### ISSUE TITLE
ci: docutils.roman module no longer exists in docutils HEAD

### ISSUE THREAD

**Issue by jayaddison** [sphinx-doc/sphinx#13124]
### Describe the bug

With a recent [modification](https://repo.or.cz/docutils.git/commit/d6e9dd44523c00ae175442b795f68542653a7b78) in the upstream `docutils` codebase (`git-svn-id: http://svn.code.sf.net/p/docutils/code/trunk@9977 929543f6-e4f2-0310-98a6-ba3bd3dd1d04`), the `docutils.roman` module has been removed, superseded by an alternative implementation in `docutils.utils._roman_numerals`.

We rely on the former to provide an implementation of `toRoman` in the Sphinx [`sphinx.writers.latex` module](https://github.com/sphinx-doc/sphinx/blob/eb337fec0acd8ce978caa0663f60f3a6afd76322/sphinx/writers/latex.py), with the [`roman` package](https://pypi.org/project/roman/) -- a non-vendored implementation of the code found in `docutils.roman`, with some updated releases since the version(s) used in `docutils` -- as a fallback.

The removal of the module causes our [LaTeX writer tests](https://github.com/sphinx-doc/sphinx/blob/eb337fec0acd8ce978caa0663f60f3a6afd76322/tests/test_writers/test_writer_latex.py) to fail, as evident in a recent [GitHub Actions `docutils` HEAD workflow failure](https://github.com/sphinx-doc/sphinx/actions/runs/11783366010/job/32820360758).

### How to Reproduce

* Clone [Sphinx 8.1.3 from source](https://github.com/sphinx-doc/sphinx/tree/a1510de4777eaa2e569435f95b05f6f3293d7035) and install it with `[test]` extras into a virtualenv.
* Clone [`docutils` from source](https://repo.or.cz/docutils.git/) and checkout the relevant commit (d6e9dd44523c00ae175442b795f68542653a7b78).
* From the virtualenv, install the locally-cloned `docutils` package (e.g. `pip install ../docutils/docutils`, or similar).
* Attempt to run the LaTeX writers test cases in Sphinx: `pytest tests/test_writers/test_writer_latex.py`

### Environment Information

```text
Platform:              linux; (Linux-6.11.5-rt-amd64-x86_64-with-glibc2.40)
Python version:        3.12.7 (main, Nov  8 2024, 17:55:36) [GCC 14.2.0])
Python implementation: CPython
Sphinx version:        8.1.3
Docutils version:      0.22b.dev
Jinja2 version:        3.1.4
Pygments version:      2.18.0
```

### Sphinx extensions

```python
N/A
```

### Additional context

N/A

Edit: clarification about the relationship between `roman` and the implementation that exists/existed in `docutils.roman`.

**Comment by jayaddison** [comment]
> The removal of the module causes our [LaTeX writer tests](https://github.com/sphinx-doc/sphinx/blob/eb337fec0acd8ce978caa0663f60f3a6afd76322/tests/test_writers/test_writer_latex.py) to fail, as evident in a recent [GitHub Actions docutils HEAD workflow failure](https://github.com/sphinx-doc/sphinx/actions/runs/11783366010/job/32820360758).

Strictly speaking this failure only occurs unless the `roman` Python package is available on the system already, since we will use that as a fallback when available: https://github.com/sphinx-doc/sphinx/blob/a1510de4777eaa2e569435f95b05f6f3293d7035/sphinx/writers/latex.py#L29-L31

**Comment by AA-Turner** [comment]
Yep, we discussed this on the DU tracker recently. I'll prepare a fix.

A

**Comment by AA-Turner** [comment]
Closed in #13131 (a pleasing PR number)
